README.md

```diff
@@ -12,10 +12,10 @@ In this application:
   |- LICENCE.md
   |- PULL_REQUEST_TEMPLATE.md
 
+```
```

Comment:
It looks like you're wrapping everything in one code block, which I would not suggest.